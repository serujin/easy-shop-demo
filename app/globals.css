@import url("https://p.typekit.net/p.css?s=1&k=gws6bus&ht=tk&f=139&a=10612664&app=typekit&e=css");
@font-face {
  font-family: "proxima-nova";
  src: url("https://use.typekit.net/af/949f99/00000000000000003b9b3068/27/l?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=n7&v=3")
      format("woff2"),
    url("https://use.typekit.net/af/949f99/00000000000000003b9b3068/27/d?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=n7&v=3")
      format("woff"),
    url("https://use.typekit.net/af/949f99/00000000000000003b9b3068/27/a?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=n7&v=3")
      format("opentype");
  font-display: auto;
  font-style: normal;
  font-weight: 700;
  font-stretch: normal;
}

* {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
  font-family: "proxima-nova";
}

html {
  color: #404040;
  font-size: 14px;
  background-color: #efefef;
}

body {
  display: flex;
  flex-direction: column;
}

header {
  display: flex;
  background-color: #4e2817;
  justify-content: center;
}

a {
  text-decoration: none;
  color: black;
}
