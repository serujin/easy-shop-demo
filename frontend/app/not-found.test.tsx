import React from 'react'
import { render } from '@testing-library/react'
import { describe, expect, it } from '@jest/globals'
import Custom404 from './not-found'

describe('404 page', () => {
  it('should render correctly', () => {
    const { container } = render(<Custom404 />)
    expect(container).toMatchSnapshot()
  })
})
