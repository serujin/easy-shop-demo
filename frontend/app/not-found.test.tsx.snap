// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page should render correctly 1`] = `
<div>
  <main
    class="main"
  >
    <div
      class="notFoundProductContainer"
    >
      <p
        class="notFoundText"
      >
        Oops, the page you are trying to reach doesn't exist
      </p>
      <p
        class="notFoundText"
      >
        Sorry for the inconveniences
      </p>
    </div>
  </main>
</div>
`;
